kolibri/core/tasks/job.py

```diff
@@ -142,12 +143,26 @@ class Job(object):
     to the workers.
     """
 
-    def __getstate__(self):
+    def to_json(self):
+        """
+        Creates and returns a JSON-serialized string representing this Job.
+        This is how Job objects are persisted through restarts.
+
+        Note: Uses DjangoJSONEncoder to encode things. When using a type that's
```

Comment:
```suggestion
```

Reply:
These comments are no longer accurate, as we have dropped use of the DjangoJSONEncoder.